map_gen/maps/HilbertSandTrap.lua

```diff
@@ -0,0 +1,287 @@
+--Hilbert Sand Trap Map, by Jayefuu, R. Nukem, and grilledham
+
+local b = require 'map_gen.shared.builders'
+local degrees = require "utils.math".degrees
+local RS = require 'map_gen.shared.redmew_surface'
+local MGSP = require 'resources.map_gen_settings'
+--https://www.fractalus.com/kerry/tutorials/hilbert/hilbert-tutorial.html
+-- Setup the scenario map information because everyone gets upset if you don't
+local ScenarioInfo = require 'features.gui.info'
+ScenarioInfo.set_map_name('Hilbert\'s Sand Trap')
+ScenarioInfo.set_map_description('You have crash landed in the middle of Hilbert\'s Labyrinth! Surrounded by quicksand and biters you must survive long enough to launch a rocket.')
+ScenarioInfo.set_map_extra_info('Only the native grasses are suitable to build on. Ores and trees have sunk into the sand, but biters have adapted to live happily in the barren landscape. Some even speak of a Hydra living deep within the desert. \n\n Map created by R. Nukem and Jayefuu, with help from grilledham and the rest of the Redmew admin team.')
+--enable Hydra
+local hail_hydra = global.config.hail_hydra
+hail_hydra.enabled = true
+-- define map settings
+local ore_settings = {
+	autoplace_controls = {
+		coal = {
+			frequency = 'very-high',
+			richness = 'normal',
+			size = 'small'
+		},
+		['copper-ore'] = {
+			frequency = 'very-high',
+			richness = 'normal',
+			size = 'normal'
+		},
+		['crude-oil'] = {
+			frequency = 'very-high',
+			richness = 'low',
+			size = 'small'
+		},
+		['iron-ore'] = {
+			frequency = 'very-high',
+			richness = 'normal',
+			size = 'normal'
+		},
+		stone = {
+			frequency = 'very-high',
+			richness = 'low',
+			size = 'small'
+		},
+		['uranium-ore'] = {
+			frequency = 'very-high',
+			richness = 'very-low',
+			size = 'very-small'
+		}
+	}
+}
+local biter_settings = {
+	autoplace_controls = {
+		['enemy-base'] = {
+			frequency = 'very-high',
+			richness = 'very-high',
+			size = 'very-high'
+		}
+	}
+}
+local tree_settings = {
+	autoplace_controls = {
+		trees = {
+			frequency = 'very-high',
+			richness = 'normal',
+			size = 'normal'
+		}
+    }
+}
+local starting_area = {
+    starting_area = 'very-small'
+}
+local water_settings = {
+	terrain_segmentation = 'high',
+    water = 'low'
+}
+--Set map settings
+RS.set_map_gen_settings(
+    {
+        MGSP.cliff_none,
+		MGSP.grass_only,
+		ore_settings,
+		biter_settings,
+		starting_area,
```

Comment:
Since you already have the presets loaded you may as well use them and save lines. :)
```suggestion
		MGSP.starting_area_very_low,
```